DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgDocumentContentRepository.cs

```diff
@@ -69,4 +73,36 @@ public int Update(DocumentContentModel entity)
                 entity.DocumentId
             });
     }
+
+    public int AddBatch(List<DocumentContentModel> models)
+    {
+        _logger.LogDebug("Inserting {count} DocumentContents in database", models.Count);
+        int rowsAffected = 0;
+        using IDbConnection connection = _connectionFactory.CreateConnection();
+        connection.Open();
```

Comment:
du åbner connection men closer aldrig? virkede det ikke med using? evt. som scope frem for statements

Reply:
Det er lidt specielt med transactions. Det fungerer stadig på samme måde (både connection og transaction bliver lukket automatisk), men man skal selv åbne connection når man er klar til at starte sin transaction